scalability_and_performance/cnf-low-latency-tuning.adoc

```diff
@@ -39,7 +39,7 @@ include::modules/cnf-allocating-multiple-huge-page-sizes.adoc[leveloffset=+2]
 
 include::modules/cnf-configure_for_irq_dynamic_load_balancing.adoc[leveloffset=+2]
 
-include::modules/ref_hardware-compatibility-with-irq-affinity.adoc[leveloffset=+3]
+include::modules/cnf-about-nic-support-irq-affinity-setting.adoc[leveloffset=+2]
 
```

Comment:
The file modules/ref_hardware-compatibility-with-irq-affinity.adoc is now unused, isn't it? Maybe you want to delete it?
Also, probably not very important but the title "about-nic-support", as we've changed the title inside the document and it doesn't reflect well what the content is, maybe it should be renamed.